packages/office-ui-fabric-react/src/components/pickers/Suggestions/Suggestions.types.ts

```diff
@@ -153,8 +153,10 @@ export interface ISuggestionsProps<T> extends React.Props<any> {
 
   /**
    * Function to fire when one of the optional remove buttons on a suggestion is clicked.
+   *
+   * TODO (adjective-object) remove IPersonaprops before the next major version bump
```

Comment:
@jdhuntington any preference for a better way to keep track of these cleanup items?

Reply:
I would leverage version control here. Push a commit to a remote that we can examine at a later date which encompasses a unit of work.